mappings/net/minecraft/network/PacketHandler.mapping

```diff
@@ -1,5 +1,8 @@
 CLASS net/minecraft/class_240 net/minecraft/network/PacketHandler
-	METHOD method_1469 (Lnet/minecraft/class_161;)V
+	METHOD method_1427 handleCloseContainer (Lnet/minecraft/class_276;)V
```

Comment:
This isn't necessarily bad, but would handleContainerClose make more sense?

Reply:
Otherwise it'd be handleSwingHand below ;)

Reply:
I applied that logic to `handleOpenScreen` -> `handleScreenOpen`